redirects/.htaccess_misc

```diff
@@ -6,6 +6,16 @@ RewriteRule ^Bloggers/Everyone$ / [R=301,L,E=nocache:1]
 # Main feed
 RewriteRule ^service/Feed/atom/Area/Bloggers(/Comments/.*)?$ /blog.atom [R=301,L,E=nocache:1]
 
+# Jay's categories
+
+RewriteRule ^Bloggers/Aerogear$ /aerogear/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/Interop$ /seam/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/Tweetstream$ /rich-faces/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/Mobile$ /rich-faces/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/Openshift$ /jay-balunas/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/Releases$ /seam/ [R=301,L,E=nocache:1]
```

Comment:
OpenShift redirecting to Jay?
Releases redirecting to Seam? That does not look right.